addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64">
```

Comment:
What about a bg-shape?

| eg |
|--------|
| ![image](https://github.com/user-attachments/assets/0a6c74f8-092d-4acd-afec-0a2d087065c2) |
| ![image](https://github.com/user-attachments/assets/4e5b26e2-46de-4485-ae8f-33418d961923) | 
| ![image](https://github.com/user-attachments/assets/bfc96b60-8b2f-4d84-85dd-7b216affb832) | 
| ![image](https://github.com/user-attachments/assets/7f1db896-5275-4693-b94e-40523343531a) |

Reply:
thanks, I added a shape and removed the list-group :)

Reply:
Thanks! I'd go with "Blob_12" tho, I think that it brings more attention to the card 😉

| this pr | proposal |
|--------|--------|
| <img width="1263" alt="image" src="https://github.com/user-attachments/assets/9d13e0f1-3db6-4d8c-b064-f2d899ef6a3d"> | ![image](https://github.com/user-attachments/assets/d87de909-d650-424d-b327-6078d3415e64) |